polling/periodic_sequence/child_workflow.go

```diff
@@ -0,0 +1,39 @@
+package periodic_sequence
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+type ChildWorkflowParams struct {
+	PollingInterval time.Duration
+}
+
+func PollingChildWorkflow(ctx workflow.Context, params ChildWorkflowParams) (string, error) {
+	logger := workflow.GetLogger(ctx)
+	logger.Info("Starting child workflow with params", params)
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout:    10 * time.Second,
+		ScheduleToCloseTimeout: 2 * time.Second,
```

Comment:
ScheduleToClose timeout should never be smaller than StartToClose timeout as it is a timeout for overall activity execution including retries. I would remove ScheduleToClose timeout here. If you want to disable retries set the activity retry options.maxAttempts to 1.